usaspending_api/references/models.py

```diff
@@ -294,6 +294,75 @@ class LegalEntity(DataSourceTrackedModel):
     business_types = models.TextField(blank=True, null=True)
     business_types_description = models.TextField(blank=True, null=True)
     recipient_unique_id = models.TextField(blank=True, null=True, verbose_name="DUNS Number")
+
+    # Business Type Categories
+    # These "aggregate" the many business type flags into buckets
+    category_business = models.BooleanField(default=False)
+    category_small_business = models.BooleanField(default=False)
+    category_other_than_small_business = models.BooleanField(default=False)
+
+    category_minority_owned_business = models.BooleanField(default=False)
+    category_alaskan_native_owned_business = models.BooleanField(default=False)
+    category_american_indian_owned_business = models.BooleanField(default=False)
+    category_asian_pacific_american_owned_business = models.BooleanField(default=False)
+    category_black_american_owned_business = models.BooleanField(default=False)
+    category_hispanic_american_owned_business = models.BooleanField(default=False)
+    category_native_american_owned_business = models.BooleanField(default=False)
```

Comment:
Yes, I'm not sure what the legal distinction is there, but they are separate categories and flags.